source/usage-and-configuration/swagger/introduction.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Swagger
+---
+
+# Swagger
+
+The theme provides tools to read and parse swagger files and generate documentations out of it.
```

Comment:
The theme provides tools to read and parse swagger files as well as generate documentation from them.

Reply:
👍